packages/components/atoms/f-button/src/components/Button.vue

```diff
@@ -28,6 +33,10 @@ export default {
         isFullWidth: {
             type: Boolean,
             default: false
+        },
+        href: {
```

Comment:
You shouldn't need to define href as a prop, as it's just a standard attribute (so it should be applied automatically onto the component when set).

Reply:
👍 removed, thanks